.github/jobs/linux.yml

```diff
@@ -66,6 +66,9 @@ jobs:
       condition: failed()
 
     - script: |
+        export DISPLAY=:99
+        Xvfb :99 -screen 0 1600x900x24 &
+        sleep 3
```

Comment:
what's this sleep for?

Reply:
AFAIK, there is no way to wait for xvfb to be ready.

Reply:
Quick search returns this: https://unix.stackexchange.com/questions/310679/how-to-poll-for-xvfb-to-be-ready